integration-tests/data/simple-circle/tree/snapcraft.yaml

```diff
@@ -0,0 +1,92 @@
+name: dep-tree
+summary: A tree of things to make sure work
+description: These don't really make sense for tests
```

Comment:
Add a longer description here for the test (comment I once got from @elopio) like

```yaml
description: |
  The objective of this test is to blah ...
  more description ... blah blah
```